src/Documentation/2.0/Migration1.5.md

```diff
@@ -12,3 +12,53 @@ The bulk of the Orleans APIs stayed unchanged in 2.0 or implementation of those
 ## Available options for configuration code
 
 ## Hosting
+
+## Logging
+Orleans 2.0 uses the same logging abstractions as ASP.NET Core 2.0. You can find replacement for most Orleans logging feature in ASP.NET Core logging. Orleans specific logging feature, such as `ILogConsumer` and message bulking, is still maintained in `Orleans.Logging.Legacy` package, so that you still have the option to use them. But how to configure your logging with Orleans changed in 2.0. Let me walk you through the process of migration.
+
+In 1.5, logging configuration is done through `ClientConfiguration` and `NodeConfiguration`. You can configure `DefaultTraceLevel`, `TraceFileName`, `TraceFilePattern`, `TraceLevelOverrides`, `TraceToConsole`, `BulkMessageLimit`, `LogConsumers`, etc through it. In 2.0, logging configuration is consistent with ASP.NET Core 2.0 logging, which means most of the configuration is done through `Microsoft.Extensions.Logging.ILoggingBuilder`. 
+
+To configure `DefaultTraceLevel` and `TraceLevelOverrides`, you need to apply [log filtering](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/logging) to `ILoggingBuilder`. For example, to set trace level to 'Information' on your grain class `MyGrain`, you can use sample below, 
+```
+siloBuilder.AddLogging(builder=>builder.AddFilter(typeof(MyGrain).FullName, LogLevel.Information));
```

Comment:
I would mention how to configure log filtering for Orleans infrastructure first, and then say "Likewise, if you use these same abstractions for your own application-level logging, you can configure the filters as follows" (but not go much further, since we should point to ASP.NET Core docs on how to *use* logging